Sources/NostrKit/Relay.swift

```diff
@@ -63,23 +66,22 @@ public struct Relay {
     
     func send(message: ClientMessage) async throws {
         return try await withCheckedThrowingContinuation { continuation in
-            print(try! message.string())
-            socket.write(string: try! message.string()) {
+            if socket.isConnected {
+                do {
+                    let message = try message.string()
+                    socket.write(string: message) {
+                        continuation.resume()
+                    }
+                } catch {
+                    continuation.resume(throwing: RelayError.writeError(error))
+                }
+            } else {
```

Comment:
Should we throw an error when resuming the continuation in case the socket is not connected?

Reply:
Yes, probably a good idea. Will update that.